docs/snippets/modules/model_io/models/llms/how_to/streaming_llm.mdx

```diff
@@ -1,5 +1,4 @@
-Currently, we support streaming for the `OpenAI`, `ChatOpenAI`, and `ChatAnthropic` implementations. To utilize streaming, use a [`CallbackHandler`](https://github.com/hwchase17/langchain/blob/master/langchain/callbacks/base.py) that implements `on_llm_new_token`. In this example, we are using `StreamingStdOutCallbackHandler`.
-
+Currently, we support streaming for the `OpenAI`, `ChatOpenAI`, `ChatAnthropic`, and `Hugging Face Text Generation Inference` implementations. To utilize streaming, use a [`CallbackHandler`](https://github.com/hwchase17/langchain/blob/master/langchain/callbacks/base.py) that implements `on_llm_new_token`. In this example, we are using `StreamingStdOutCallbackHandler`.
```

Comment:
There are others too, e.g. Replicate was added in https://github.com/hwchase17/langchain/pull/8045 and if you just search for "if self.streaming" in the relevant source folder you will see a few others also support streaming e.g. tongyi and jinachat.

So perhaps we should just update this comment and say something like "Currently, streaming is supported for a subset of LLM implementations" to keep it generic.

Ideally, all/most LLM implementations should support streaming eventually (soon)

Reply:
Ya I thought of that too. Could we add examples for each?